sk0-front/src/app/auth/store/auth.effects.ts

```diff
@@ -60,6 +62,11 @@ export class AuthEffects {
       mergeMap((action) =>
         this.authService.register(action).pipe(
           map((response) => registerSuccess(response)),
+          tap(async () => {
+            await this.notificationService.alertSuccess(
+              "Megerősítő email elküldve. Nézze meg az email fiókját."
+            );
+          }),
```

Comment:
This is cool for now, be we definitely can separate it later into a notificationEffects too. And decouple it from the auth effects, in that case it only would listen to the `registerSuccess` effects.